HW_16/TODO/list.php

```diff
@@ -0,0 +1,91 @@
+<?php
+declare(strict_types=1);
+
+error_reporting(E_ALL);
+ini_set('display_errors', '1');
+
+require_once __DIR__ . '/db.php';
+require __DIR__ . '/header.php';
+
+if(session_status() != 'PHP_SESSION_ACTIVE'){
+    session_start();
+}
+
+$conn = getConnection();
+ $id = $_SESSION['current_user']['id'];
+
+$query_lists = "SELECT a.id, a.id_users, a.name, created_at, a.is_done FROM lists a where a.id_users = '$id'";  //where b.id_users = '$userIDd'
+
+$lists = mysqli_query($conn, $query_lists);
+require __DIR__ . '/footer.php';
+?>
+<nav class="navbar navbar-expand-lg bg-light">
+    <div class="container">
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                <li class="nav-item dropdown">
+                    <a class="dropdown-item" href="/php_study/TODO/add.php">Add</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+</nav>
+
+<div class="container">
+<table class="table table-striped">
+    <thead>
+    <tr>
+        <th>#</th>
+        <th>Done</th>
+        <th>TODO</th>
+        <th>Created At</th>
+        <th>Delete</th>
+    </tr>
+    </thead>
+    <tbody>
+    <?php if ($lists): ?>
+        <?php foreach ($lists as $list): ?>
+            <tr>
+                <th><?= $list['id'] ?></th>
+                <td>
+                    <div class="form-check">
+                        <form action="/php_study/TODO/update.php" method="post" class="d-inline" id="my-form">
+                            <input class="form-check-input" type="checkbox" onchange="myFunction()" id="flexCheckDiseble"
+                                <?php if ($list['is_done']):?>
+                                    checked
+                                <?php endif; ?>>
+                        </form>
```

Comment:
добавь сюда input со скрытым id, и на беке ты уже сможешь получить id дела из POST
```suggestion
                            <input type="hidden" name="id" value="<?= $list['id'] ?>">
                        </form>
```